client/Application.jsx

```diff
@@ -98,19 +101,35 @@ class App extends React.Component {
         this.props.receiveLobbyMessage({});
     }
 
+    getUrlParameter(name) {
+        name = name.replace(/[[]/, '\\[').replace(/[\]]/, '\\]');
+        var regex = new RegExp('[\\?&]' + name + '=([^&#]*)');
+        var results = regex.exec(location.search);
+        return results === null ? '' : decodeURIComponent(results[1].replace(/\+/g, ' '));
+    }
+
     render() {
         var rightMenu = this.props.loggedIn ? authedMenu : notAuthedMenu;
         var component = {};
 
         var path = this.props.path;
         var pathArg = undefined;
+        var idArg = undefined;
+        var tokenArg = undefined;
         var index = path.indexOf('/decks/edit');
 
         if(index !== -1) {
             path = path.substr(index, 11);
             pathArg = this.props.path.substr(11 + 1);
         }
 
+        index = path.indexOf('/reset-password');
+        if(index !== -1) {
+            path = path.substr(index, 15);
+            idArg = this.getUrlParameter('id');
+            tokenArg = this.getUrlParameter('token');
```

Comment:
Nevermind, it seems that this is part of react-router which isn't being used (and has since changed to `getQuery()` anyway)